CRM/Core/SelectValues.php

```diff
@@ -774,6 +774,7 @@ public static function getSubscriptionHistoryMethods() {
     return [
       'Admin' => ts('Admin'),
       'Email' => ts('Email'),
+      'Form' => ts('Form'),
```

Comment:
@colemanw just wondering is there a reason not to use Web ?

Reply:
IMO it's good to add more distinction. I'd actually like to include the full name of the form as the 'method', but that would take more work, so for now... this.